From a8bbf4bd438e320f5ac7e4622d2aba48315aa794 Mon Sep 17 00:00:00 2001
From: Andreas Cord-Landwehr <cordlandwehr@kde.org>
Date: Sun, 13 Oct 2024 08:42:48 +0200
Subject: [PATCH] Remove ASPELL runtime dependency from plugin building check

The ASPELL executable is only required at runtime. For plugin building
reduce check to only include dir and libraries.

Upstream-Status: Submitted [https://invent.kde.org/frameworks/sonnet/-/merge_requests/112]

SPDX-FileCopyrightText: Andreas Cord-Landwehr <cordlandwehr@kde.org>
SPDX-License-Identifier: MIT
Signed-off-by: Andreas Cord-Landwehr <cordlandwehr@kde.org>
---
 src/plugins/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/CMakeLists.txt b/src/plugins/CMakeLists.txt
index b53c9de7..cf35f896 100644
--- a/src/plugins/CMakeLists.txt
+++ b/src/plugins/CMakeLists.txt
@@ -8,7 +8,7 @@ set_package_properties(ASPELL PROPERTIES
     DESCRIPTION "Spell checking support via Aspell"
     TYPE OPTIONAL
 )
-if (ASPELL_FOUND)
+if (ASPELL_INCLUDE_DIR AND ASPELL_LIBRARIES)
    add_subdirectory( aspell )
 endif ()
 
-- 
2.43.0

